resolvers/Payment_Gateway/checkout.js

```diff
@@ -0,0 +1,148 @@
+const stripe = require('stripe')('PUT YOUR SECURITY KEY HERE');
+const express = require("express");
+const router = express.Router();
+const { resolve } = require('path');
+router.post("/payment", 
+	[
+		check('amount','amount is not in the body').not().isEmpty(),
+		check('id','The method of payment is not present').not().isEmpty()
+	],
+	async (req, res) => {
+		let {amount, id} = req.body //here amount is the amount for donation, and id is the way the user is going to pay like card UPI and other methods
+		try {
+			const payment = await stripe.paymentIntents.create({
+				amount,
+				currency: "USD",//Put your currency here,
+				description: "description",//put your description here,
+				payment_method: id,
+				confirm: true
+			});
+			console.log("Payment", payment)
+			res.json({
+				message: "Payment successful",
+				success: true
+			})
+		} catch (error) {
+			console.log("Error", error)
+			res.json({
+				message: "Payment failed",
+				success: false
+			})
+		}
+	});
+	// Copy the .env.example in the root into a .env file in this folder
+	require('dotenv').config({ path: './.env' });
+	
+	// Ensure environment variables are set.
+	checkEnv();
+	
+	const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
```

Comment:
const { STRIPE_SECRET_KEY } = require('config');

Reply:
Config file must have all mappings of env vars: 

module.exports = { 
    STRIPE_SECRET_KEY: process.env.STRIPE_SECRET_KEY
};